src/components-examples/angular/menu/menu-grouping/menu-grouping-example.html

```diff
@@ -0,0 +1,10 @@
+<button [sbbContextmenuTriggerFor]="menu" aria-label="Show stations" type="button"></button>
+<sbb-menu #menu="sbbMenu">
+  <span class="sbb-label" id="stations-bern">Railway stations in Bern</span>
+  <button sbb-menu-item type="button" aria-describedby="stations-bern">Bern HB</button>
+  <button sbb-menu-item type="button" aria-describedby="stations-bern">Bern Wankdorf</button>
+  <span class="sbb-label" id="stations-zurich">Railway stations in Zurich</span>
+  <button sbb-menu-item type="button" aria-describedby="stations-zurich">Zurich HB</button>
```

Comment:
I think this solution could be ok. what I originally meant, was that you keep your div with the role list, but you set aria-describedby on the list.

Reply:
Now I understand why the screenreader reads every label again :)

Reply:
Okay, sorry. This was a hard one 😄